config.contrib.yaml

```diff
@@ -1,6 +1,8 @@
 #ddev-generated
 ## Command provided by https://github.com/ddev/ddev-drupal-contrib
 web_environment:
+  # If desired, override to a different version of Drupal core in via the project's DDEV config
+  - _TARGET_CORE=^10
```

Comment:
IMO this env name would be better and clearer with the word `DRUPAL` in it, maybe `_TARGET_DRUPAL_CORE`.

Reply:
I agree, but this var is read by code that we dont create so I'll just accept the ambiguity.